test/uri/test_ws.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: false
+require 'test/unit'
+require 'uri/ws'
+
+module URI
+
+
+class TestWS < Test::Unit::TestCase
```

Comment:
Is this indented correctly?

Reply:
I followed the indenting pattern for the existing tests, such as in [test_http.rb](https://github.com/ruby/uri/blob/master/test/uri/test_http.rb) and [test_ldap.rb](https://github.com/ruby/uri/blob/master/test/uri/test_ldap.rb) etc.

Reply:
This convention is bogus, you should just indent it as per normal IMHO :)